docs/user-guide.rst

```diff
@@ -413,6 +413,19 @@ configuration like that:
       enabled: true
       externalScrapeTargetsOnly: true
 
+
+API Server
+~~~~~~~~~~~
+
+It is possible to let Tarmak create an public endpoint for your APIserver.
+This can be used together with `Secure public endpoints <user-guide.rst#secure-public-endpoints>`_.
```

Comment:
This link isn't going to work on the tarmak docs website AFAIK (and doesn't work for me locally either)